### ISSUE TITLE
Roassal2 on VisualWorks 9.2?

### ISSUE THREAD

**Issue by mrobe** [ObjectProfile/Roassal2#67]
Trying to get it to work on macOS 12.5 (Intel), but I keep getting UHEs at #nsBitmap.

The bundle comment for Roassal2 says:

> On Windows (7 through 10), this version of Roassal requires Cairo library version 1.10.2 (provided in VisualWorks 8.3 and after) and does not work with version 1.14.2.  Both versions of the Cairo library are provided in VisualWorks 8.3.1 and after, with the relevant ExternalInterface class definition data set to see the older version.  With a small code change, a knowledgeable user can move to the new version and load a later compatible version of Roassal from the Cincom Open repository, should they so wish.

There are several issues here:

(1) brew cannot find Cairo 1.10.x

(2) The library isn't provided in VisualWorks, as the bundle comment says (unless 'library' actually means 'parcel' and it's "OpenGL-CairoGraphics"...?)

(3) The "small code change" isn't explained.

(4) Most of the packages don't have comments.

How to make this work?

**Comment by akevalion** [comment]
Hi @mrobe, I do not use VW for 4 years from now, but I remember that when you wanted to use roassal2 with VW, you in the installation process of VW, you should load and select the cairo graphics package to used, if not, you will not be able to install it.

Buf for now my current focus ir Roassal3(https://github.com/ObjectProfile/Roassal3), let me know you are interested on have roassal3 on VW

**Comment by mrobe** [comment]
Hi @akevalion, thanks for your speedy reply. I tried the VisualWorks Installer again to get the Cairo libraries and it works! I can now run the various examples.

It might be a good idea to mention this step in the bundle comment.

I've only just started with Roassal, but if Roassal3 were available for VW, I'd be happy to try it.

EDIT: (...especially if Roassal3 includes candlestick charts!)

Thanks again!
